core/tests/webdriverio_utils/forms.js

```diff
@@ -439,6 +439,8 @@ var MultiSelectEditor = function(elem) {
     },
     expectCurrentSelectionToBe: async function(expectedCurrentSelection) {
       // Open the dropdown menu.
+      var searchBarDropdownToggleElement = elem.$(
+        '.e2e-test-search-bar-dropdown-toggle');
```

Comment:
Why was this added?

Reply:
It was not added by mistake when I migrated this dependency, it is same as we have in protractor. PTAL [here](https://github.com/oppia/oppia/blob/develop/core/tests/protractor_utils/forms.js#L395)

Reply:
Ditto above.

Reply:
Same as above.